.github/stale.yml

```diff
@@ -4,7 +4,7 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 90
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 90
+daysUntilClose: 7
```

Comment:
Maybe we can use the default value of slate bot which is 21 days? The bigger problem here is many issues are not slate anymore when they reopen.
https://github.com/etcd-io/etcd/blob/main/.github/stale.yml#L9

Reply:
We can always use `/lifecycle frozen` for important issues that needs more time, but the fact is that we currently have [~310 issues](https://github.com/kubeflow/pipelines/labels?sort=count-desc) with `lifecycle/stale` and ~90 issues with `lifecycle/frozen`. This number is already unmanageable for the number of contributors we have.

Let me know your thoughts.

Reply:
As for the "many issues are not slate anymore when they reopen", we can also consider setting daysUntilStale to the default 60 days.

Reply:
Thanks @rimolive, shorten the `daysUntilStale` and having a longer `daysUntilClose` would work better. This way we can warn the issue creator on issues going to stale without closing it too soon.

Reply:
Let me know if these parameters look good:
`daysUntilStale`: 60
`daysUntilClose`: 21